pkg/cmd/prebuild/add.go

```diff
@@ -134,8 +179,10 @@ var prebuildAddCmd = &cobra.Command{
 	},
 }
 
-var runOnAddFlag bool
-
 func init() {
 	prebuildAddCmd.Flags().BoolVar(&runOnAddFlag, "run", false, "Run the prebuild once after adding it")
+	prebuildAddCmd.Flags().StringVar(&branchFlag, "branch", "", "Git branch for the prebuild")
```

Comment:
Add shorthands (e.g. `StringVarP`) for all flags except `run`

`branch` => `b`
`retention` => `r`
`commit-interval` => `c`
`trigger-files` => `t`

Reply:
modified it .